new icon: cmd (original, plain, line)

gewaleelek opened the pull request:
## Double check these details before you open a PR

<!-- Tick the checkboxes to ensure you've done everything correctly -->
- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/wiki/Overview-on-Submitting-Icons)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/wiki/Organizing-SVGs)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/wiki/SVG-Standards)
- [x] A new object is added in the `devicon.json` file at the correct alphabetic position as seen [here](https://github.com/devicons/devicon/wiki/Updating-%60devicon.json%60)

<!-- Refer to the [contributing](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#contributing-to-devicon) guidelines for more details. -->

**This PR closes #539**
<!-- List issues that this PR would close above. Ex: This PR closes #1, #2, #3. -->

## Link to prove your SVG is correct and up-to-date.
<!-- Link to an official page/wiki goes here. Anything that proves your SVGs are the correct ones. -->
[Wikipedia](https://en.wikipedia.org/wiki/Cmd.exe#)

gewaleelek committed:
new icon: cmd (original, plain, line)

canaleal commented:
Icon looks great!

Just want to confirm if this is the official icon.

gewaleelek commented:
The SVG itself is my attempt at copying the look of the cmd.exe icon shown on the Wikipedia page. Let me know if I should use something else.

Snailedlt commented:
Microsoft products are notoriously bad at branding. I don't think cmd has any consistent branding, but if anyone knows it might be the powershell team (ref: https://github.com/devicons/devicon/issues/364#issuecomment-1158979222)

Snailedlt committed:
Merge branch 'develop' into cmd

canaleal commented:
Closing as it's been 2 months without update. Can reopen ticket if there is new info.

gewaleelek commented:
@Snailedlt @canaleal 

Hello, I've extracted the icon of `cmd.exe` in a Windows 11 machine using this [PowerShell script](https://gist.github.com/otonii/4fdba541c865513e14637b99c9375928), and converted it to a png so it can be displayed here. Do you two think this counts as "official" or do I need more sources than this still?

<img width="32" height="32" alt="cmd" src="https://github.com/user-attachments/assets/339e874e-b64a-4759-9568-dc9d15d6689e" />

I can update the icons and submit a new PR if you think it's okay to use that.
